pkg/scheduler/apis/config/types.go

```diff
@@ -96,6 +96,11 @@ type KubeSchedulerConfiguration struct {
 	// Extenders are the list of scheduler extenders, each holding the values of how to communicate
 	// with the extender. These extenders are shared by all scheduler profiles.
 	Extenders []Extender
+
+	// MaintainCacheNonLeader enables caching for non-leader candidates. This makes non-leader candidates
+	// start with watching cache. Doing so will have faster failover at cost of higher memory overhead.
+	// Defaults to true.
+	MaintainCacheNonLeader bool
```

Comment:
`MaintainCacheInNonLeader` or `MaintainCacheWhenNonLeader` is more readable to me. wdyt?

Reply:
`MaintainCacheWhenNonLeader` sounds better.

Reply:
Done